config/config.go

```diff
@@ -91,6 +93,55 @@ type (
 	}
 )
 
+// MetricsLevelValue is a custom kingpin.Value that parses metrics levels directly into metrics.Level
+type MetricsLevelValue struct {
+	level *metrics.Level
+}
+
+// NewMetricsLevelValue creates a new MetricsLevelValue with the given target
+func NewMetricsLevelValue(target *metrics.Level) *MetricsLevelValue {
+	return &MetricsLevelValue{level: target}
+}
+
+// Set implements kingpin.Value interface - parses and accumulates metrics levels
+func (m *MetricsLevelValue) Set(value string) error {
+	// Parse the single value into a level
+	level, err := metrics.ParseLevel([]string{value})
+	if err != nil {
+		return err
+	}
+
+	// If this is the first value and it's not a special "all" or "none" case,
+	// initialize to none first to clear any default
+	if *m.level == metrics.MetricsLevelAll && value != "all" {
+		*m.level = metrics.MetricsLevelNone
+	}
+
+	// Handle special cases
+	if value == "all" {
+		*m.level = metrics.MetricsLevelAll
+		return nil
+	}
+	if value == "none" {
+		*m.level = metrics.MetricsLevelNone
+		return nil
+	}
```

Comment:
Lets avoid the none and all cases until there is a specific use case.

Reply:
i don't see the need to remove these, the meanings are quite clear.

Reply:
the argument against all is similar. By default all metrics are exported, so 
```
metricLevels: [] | null 
``` 
exports all metrics as well as ... 

```
metricLevels: 
 - all
```

the `all` example looks odd IMHO.